scripts/test/launchTestUtilities.ts

```diff
@@ -91,6 +91,7 @@ async function getVSCodeCliArgs(params: {
         extensionTestsEnv: {
             ['DEVELOPMENT_PATH']: projectRootDir,
             ['AWS_TOOLKIT_AUTOMATION']: params.suite,
+            ...(params.suite === 'e2e' && { ['E2E_TESTING_DIRECTORY']: process.env.E2E_TESTING_DIRECTORY }),
```

Comment:
random special cases buried in the codepath should be avoided. `getVSCodeCliArgs` is a general util. the other env vars here are driven by parameters, yet now we have a random special case in the middle of this function. that makes it harder to reason about the flow.

Reply:
relativeTestEntryPoint just points to the test runner script, I don't think we want that part configurable since that's the thing that actually loads all the tests